Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.html

```diff
@@ -125,15 +146,33 @@
                   <mat-icon>info_outline</mat-icon>
                   <span i18n="@@NavigationComponentInformation">Information</span>
                 </button>
+                <button mat-menu-item>
+                  <mat-icon>new_releases</mat-icon>
+                  <span
+                    matBadge="!"
+                    matBadgeColor="accent"
+                    matBadgeSize="small"
+                    [matBadgeHidden]="!releaseInfoService.newUpdate"
+                    (click)="releaseInfoService.displayReleaseDialog()"
+                    i18n="NavigationComponent|Release Notes Menu Item@@NavigationComponentReleaseNotes"

```

Comment:
```suggestion
                    i18n="@@NavigationComponentReleaseNotes"
```

Reply:
@mschwrdtnr just so you know on consecutive translations you can just put the ID of the translation for brevity.